tests/src/core/physics/Joint.test.ts

```
@@ -5,334 +5,309 @@
 
-    // Test that damping works correctly.
-    let tempDamping = joint.damping;
-    joint.damping = tempDamping;
-    expect(joint.damping).to.eq(tempDamping);
+  it("connectedInertiaScale", function () {
+    engine.sceneManager.activeScene.physics.gravity = new Vector3(0, 0, 0);
+    const box1 = addBox(new Vector3(1, 1, 1), DynamicCollider, new Vector3(0, 1, 0));
+    const box2 = addBox(new Vector3(1, 1, 1), DynamicCollider, new Vector3(1, 1, 0));
+    const fixedJoint = box1.addComponent(FixedJoint);
+    const box1Collider = box1.getComponent(DynamicCollider);
+    const box2Collider = (fixedJoint.connectedCollider = box2.getComponent(DynamicCollider));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)